### DISCUSSION TITLE
Xournalpp built on Raspberry Pi OS

### DISCUSSION THREAD

**Discussion by ThierryM** [xournalpp/xournalpp#2503] [Show and tell] [answered]
Hi,
I've compiled Xournal++ under a Raspberry Pi 4 (under Raspberry Pi OS) following the Debian/Ubuntu instructions.
I had to install the "gettext" package in order to build with no message error and the translated files (the "gettext-base" package was already here).
Maybe, the [documentation page about the Linux build](https://github.com/xournalpp/xournalpp/blob/master/readme/LinuxBuild.md) should be completed.
Regards,

Thierry

**Comment by LittleHuba** [comment, marked as answer]
Please provide a PR with the updated instructions.

**Reply by ThierryM** [reply]
Hi,
PR made.
Regards.
